webcrypto/subtle_crypto.go

```diff
@@ -36,73 +37,72 @@ type SubtleCrypto struct {
 // The `key` parameter should be a `CryptoKey` to be used for encryption.
 //
 // The `data` parameter should contain the data to be encryption.
-func (sc *SubtleCrypto) Encrypt(algorithm, key, data goja.Value) *goja.Promise {
+func (sc *SubtleCrypto) Encrypt( //nolint:dupl // we have two similar methods
+	algorithm, key, data goja.Value,
+) *goja.Promise {
 	rt := sc.vu.Runtime()
-	promise, resolve, reject := promises.New(sc.vu)
 
-	// 2.
-	// We obtain a copy of the key data, because we might need to modify it.
-	plaintext, err := exportArrayBuffer(rt, data)
-	if err != nil {
-		reject(err)
-		return promise
-	}
+	var plaintext []byte
+	var ck CryptoKey
+	var encrypter EncryptDecrypter
```

Comment:
I am suprised gofmt/gofumpt doesn't reformat this to a block of var :thinking:

Reply:
But it's good to do that. That way, the "shared" variables will be in the same logical block, going to adjust :+1:

Reply:
Updated with the latest force push